### DISCUSSION TITLE
Issues with Triple Display

### DISCUSSION THREAD

**Discussion by octomodz** [missionpinball/missionpinball#6] [General]
I did a fresh install of windows and MPF recently for my League of Legends pinball. GOt everything going again but the triple display that was previously;y working is now only displaying on the main monitor. It shows the 3 displays that are meant to go to across the monitors on the main. Here is the code that previously worked

window:
  width: 2560
  height: 3000
  title: League of Legends
  fullscreen: true
  resizable: true
  borderless: true
  exit_on_escape: true
  no_window: false
  show_cursor: no 

displays:

  window:
    width: 2560
    height: 3000
  display1:
    width: 2560
    height: 1440
  display2:
    width: 1920
    height: 1080
  display3:
    width: 800
    height: 480

slides:
  slide_layout:
    target: window
    background_color: white #white for test
    widgets: 
      - type: display
        source_display: display1
        width: 2560
        height: 1440
        x: right
        y: 2400
        anchor_x: right

      - type: display
        source_display: display2
        width: 1920
        height: 1080
        x: left
        y: 1140
        anchor_x: left
      - type: display
        source_display: display3
        width: 800
        height: 480
        x: left
        y: 125
        anchor_x: left
        anchor_y: bottom 

slide_player:
  mc_ready:
    slide_layout:
      target: window

**Comment by colemanomartin** [comment]
There are a couple system problems I would rule out, which I suspect you already checked. If your extended desktop is not the resolution MPF expects, the multiple display will not work. Is the second and third monitors powered up when MPF starts? If your extended desktop monitor positions are different, it could also cause this problem. Also, if your MPF window is not the size of the extended desktop, (your MPF window is only full screen to your main monitor) this problem can happen. My config has a couple extra settings declared.  I'll attach it in case its useful to you. 

kivy_config:
  kivy: # https://kivy.org/doc/stable/api-kivy.config.html#available-configuration-tokens
    desktop: 1 # use 1. This option controls desktop OS specific features, such as enabling drag-able scroll-bar in scroll views, disabling of bubbles in TextInput etc. 0 is disabled, 1 is enabled.
    exit_on_escape: true # Enables exiting kivy when escape is pressed. 0 is disabled, 1 is enabled.
    pause_on_minimize: 0 # If set to 1, the main loop is paused and the on_pause event is dispatched when the window is minimized. This option is intended for desktop use only. Defaults to 0.
  graphics:  # https://kivy.org/doc/stable/api-kivy.config.html#available-configuration-tokens
    borderless: true # If set to True, removes the window border/decoration. Window resizing must also be disabled to hide the resizing border.
    window_state: visible # and maximized allows it to exist in a window. Visible = whole screen visible, also valid: hidden, minimized
    fbo: hardware  # uese hardware # also valid software, force-hardware
    left: 0
    top: 0
    width: 3840 # 4K monitor is 3840x2160 Playfield jpg is 508x1190
    height: 3184 #  2160+1024 of pf monitor
    maxfps: 30
    multisamples: 2
    position: custom # use custom # auto, custom
    show_cursor: false #use false
    rotation: 0
    resizable: false #use false when true after title bar grabbed, the window can be resized. false=no resizing
    fullscreen:  true #use true leaves a title bar which can be grabbed allowing display to be windowed. False no title bar, no grabbing, fake
